bot_components/slack_comm.py

```diff
@@ -8,7 +8,7 @@
 """
 import json
 
-import bot_components
+from .. import bot_components
```

Comment:
I think some of this might have to do with the semi-strange way these are installed as their own packages rather than being all part of one package.

I skipped the `pip install` for the package directory and instead relied on them being in the plugins directory for rtmbot in which case the entire directory is treated as a single package.